types/node/index.d.ts

```diff
@@ -2695,6 +2697,10 @@ declare module "dgram" {
         setMulticastLoopback(flag: boolean): void;
         addMembership(multicastAddress: string, multicastInterface?: string): void;
         dropMembership(multicastAddress: string, multicastInterface?: string): void;
+        getRecvBufferSize(): number;
+        getSendBufferSize(): number;
+        setRecvBufferSize(size: number): void;
+        setSendBufferSize(size: number): void;
```

Comment:
These APIs are already present (see #20535).

Reply:
ok, this proves I can't read or use grep ☹️

Reply:
Or git merges without thinking too much. The PR I referred to was merged last night.